documentation/staging/content/samples/simple/azure-kubernetes-service/includes/create-aks-cluster-storage.txt

```diff
@@ -137,6 +137,53 @@ spec:
   - nobrl
 ```
 
+**Note:** This sample includes `nobrl` in the `mountOptions` to disable byte range file locking on the `azurefile` storage class.  This is necessary as of this writing because the `azurefile` storage class does not support advisory byte range locking.  This approach is documented in the [Azure Kubernetes Service FAQ](https://docs.microsoft.com/en-us/azure/aks/troubleshooting#what-are-the-default-mountoptions-when-using-azure-files).
```

Comment:
This is necessary as of this writing because -> Currently, this is necessary because

Reply:
Fixed in 9e2dc7cfc.